### DISCUSSION TITLE
Is it possible to run this library on laravel 9 but with property_id in config

### DISCUSSION THREAD

**Discussion by ahsanf** [freekmurze/laravel-analytics#483] [Q&A] [answered]
Is it possible to run this library on laravel 9 but with property_id in config? because what I see now for property_id is only on version 5.0.0, and that version requires laravel to be on version 10. Even though view_id (which is on version 4.1.1) is already no more in google analytics

**Comment by parabellumKoval** [comment]
I have a similar question. @ahsanf do you have a solution?

**Comment by timvandijck** [comment, marked as answer]
If updating Laravel is not an option you might look into forking the package and changing the composer.json to your needs. Although the package does not support Laravel versions before 10, I don't think there's a lot of specific L10 code in it, so this should not be too hard.
